src/modules/SignIn.js

```diff
@@ -0,0 +1,122 @@
+import { Formik } from "formik";
+import { useHistory } from "react-router-dom";
+import FormInput from '../FormInput';
+import FormButton from '../FormButton';
+
+
+function SignIn() {
+    const initialValues = {
+        email: '',
+        password: '',
+    };
+
+    let validate = (values) => {
```

Comment:
this looks good, however, maybe you could use Yup for validation?

Reply:
Ok, I switched to Yup